cmd/metrics-v3-cache.go

```diff
@@ -24,6 +24,10 @@ import (
 	"github.com/minio/minio/internal/cachevalue"
 )
 
+// initialDriveIOStats is used to calculate "per second"
+// values for IOStat related disk metrics e.g. reads/sec.
+var initialDriveIOStats *diskIOStats
```

Comment:
It seems like this is set on first request and then never updated again, so this will give the lifetime average of these values? Correct?

If we are just interested in the lifetime you should just initialize it right away and not risk a race.

IMPORTANT Question is if it is lifetime average we are interested in - or a rolling average for a specific time window?

Lifetime becomes increasingly less useful the longer the server is running. And if a server has been running for 2 months, it will take days for an issue to even become visible here.

A rolling sum over last minute would make more sense to me.

Reply:
Thanks @klauspost , have updated to use "last minute" data instead of lifetime values.